src/router.tsx

```diff
@@ -168,8 +168,8 @@ function AppRouter() {
               element={<RedirectToRobot />}
             />
 
-            <Route path="hero/:address/" element={<ValidatorsDetails />} />
-            <Route path="hero/:address/:tab" element={<ValidatorsDetails />} />
+            {/* <Route path="hero/:address/" element={<ValidatorsDetails />} />
```

Comment:
important to add redirects, if some external links exists
check `src/routing/Brain.tsx`